### PR TITLE
Add speech api

### PR DESCRIPTION
**Pull request by lickem22**
Reviewer: @sidravi1 
Estimate: 20 mins

---

## Ticket

Fixes: JIRA_TICKET_LINK

### Goal
The MomConnect team is interested in testing the speech api. In order to make it easier, we need an action to create an image after release.
### Changes
- Created a github action to push an image of the speech api.
## How has this been tested?
-  Testes the action manually
## Checklist

Fill with `x` for completed. 

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my own code to ensure good quality
- [x] I have tested the functionality of my code to ensure it works as intended
- [x] I have resolved merge conflicts

(Delete any items below that are not relevant)
- [ ] I have updated the automated tests
- [ ] I have updated the scripts in `scripts/`
- [ ] I have updated the requirements
- [ ] I have updated the README file
- [ ] I have updated affected documentation
- [ ] I have added a blogpost in Latest Updates
- [x] I have updated the CI/CD scripts in `.github/workflows/`
- [ ] I have updated the Terraform code

### CODE REVIEW

File: .github/workflows/build_push_core_backend.yaml [hunk 1]
```diff
@@ -37,8 +37,8 @@ jobs:
         uses: aws-actions/amazon-ecr-login@v2
         with:
           registry-type: public
-      
-      - name: Build and push Core Backend
+
+      - name: Build and push Speech api
```
Comment (sidravi1) [inline .github/workflows/build_push_core_backend.yaml:41]: Should this be "Core Backend"?

File: .github/workflows/build_push_speech_api.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          role-session-name: github-actions
+          aws-region: us-east-1 # ${{ vars.AWS_REGION }}
+
+      - name: Login to Amazon ECR
+        id: login-ecr-public
+        uses: aws-actions/amazon-ecr-login@v2
+        with:
+          registry-type: public
+
+      - name: Build and push Core Backend
```
Comment (sidravi1) [inline .github/workflows/build_push_speech_api.yaml:42]: Should this say "Speech API"?

**Review by sidravi1 — APPROVED** [review]
Couple of minor typos to fix but looks good.

Thanks @lickem22!

File: .github/workflows/build_push_core_backend.yaml (hunk 1 above)
Comment (lickem22) [reply .github/workflows/build_push_core_backend.yaml]: Yes interverted it.

**Pull request merged**